### PR TITLE
Expose 9945 for checking relaychain's block number on parachain

### PR DESCRIPTION
**Pull request by Dengjianping**
Signed-off-by: Dengjianping <djptux@gmail.com>

## Description

~~Currently seems docker cannot parse bootnodes in `ENTRYPOINT`.~~
~~Originally, `ENTRYPOINT` in dockerfile like this:~~
```
ENTRYPOINT [\
  "/usr/local/bin/manta",\
  "--chain",\
  "/usr/share/calamari.json",\
  "--bootnodes",\
  "/dns/crispy.calamari.systems/tcp/30333/p2p/12D3KooWNE4LBfkYB2B7D4r9vL54YMMGsfAsXdkhWfBw8VHJSEQc", \
  "/dns/crunchy.calamari.systems/tcp/30333/p2p/12D3KooWL3ELxcoMGA6han3wPQoym5DKbYHqkWkCuqyjaCXpyJTt", \
  "/dns/hotdog.calamari.systems/tcp/30333/p2p/12D3KooWMHdpUCCS9j8hvNLTV8PeqJ16KaVEjb5PVdYgAQUFUcCG", \
  "/dns/tasty.calamari.systems/tcp/30333/p2p/12D3KooWGs2hfnRQ3Y2eAoUyWKUL3g7Jmcsf8FpyhVYeNpXeBMSu", \
  "/dns/tender.calamari.systems/tcp/30333/p2p/12D3KooWNXZeUSEKRPsp1yiDH99qSVawQSWHqG4umPjgHsn1joci" \
]
```
~~Previously it worked fine.~~

~~But now, it failed to parse bootnodes in `ENTRYPOINT`. So our CI always gets error on docker image check.~~
~~Removing them, it worked.~~

Expose port `9945` to improve CI on checking relaychain's blocks.

closes: #XXXX

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (`manta` or `dolphin`) with right title (start with [Manta] or [Dolphin]),
- [ ] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [ ] Wrote unit tests.
- [ ] Updated relevant documentation in the code.
- [ ] Added **one** line describing your change in `<branch>/CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] If runtime changes, need to update the version numbers properly:
   * `authoring_version`: The version of the authorship interface. An authoring node will not attempt to author blocks unless this is equal to its native runtime.
   * `spec_version`: The version of the runtime specification. A full node will not attempt to use its native runtime in substitute for the on-chain Wasm runtime unless all of spec_name, spec_version, and authoring_version are the same between Wasm and native.
   * `impl_version`: The version of the implementation of the specification. Nodes are free to ignore this; it serves only as an indication that the code is different; as long as the other two versions are the same then while the actual code may be different, it is nonetheless required to do the same thing. Non-consensus-breaking optimizations are about the only changes that could be made which would result in only the impl_version changing.
   * `transaction_version`: The version of the extrinsics interface. This number must be updated in the following circumstances: extrinsic parameters (number, order, or types) have been changed; extrinsics or pallets have been removed; or the pallet order in the construct_runtime! macro or extrinsic order in a pallet has been changed. If this number is updated, then the spec_version must also be updated 
- [ ] If needed, notify the committer this is a draft-release and a tag is needed after merging the PR.
- [ ] Verify benchmarks & weights have been updated for any modified runtime logics
- [ ] If needed, bump `version` for every crate.
- [ ] If import a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- [ ] If needed, update our Javascript/Typescript APIs. These APIs are offcially used by exchanges or community developers.
- [ ] If we're going to issue a new release, freeze the code one week early(it depends, but usually it's one week), ensure we have enough time for related testing.
- [ ] Check if inheriting any upstream runtime storage migrations. If any, perform tests with `try-runtime`.

### CODE REVIEW

**Review by ghzlatarev — APPROVED** [review]
I'm fine with this workaround, if no one can find a solution.

**Comment by Dengjianping** [comment]
> 

I think we should move bootnodes to genesis. It's not that convenient for node runners who have to append bootnodes while a node is started.

**Comment by Dengjianping** [comment]
I built image with the new genesis, it works fine, and can sync blocks from mainnet. 
`bootNodes` doesn't affect the sha of genesis.

**Review by grenade — APPROVED** [review]
lgtm

**Comment by grenade** [comment]
the hotdog node key is actually `12D3KooWBdto53HnArmLdtf2RXzNWti7hD5mML7DWGZPD8q4cywv`

File: docker/calamari.Dockerfile [hunk 1]
```diff
@@ -33,16 +33,6 @@ ADD $PARA_GENESIS_URL $PARA_GENESIS_PATH
...
-  "/usr/local/bin/manta",\
-  "--chain",\
-  "/usr/share/calamari.json",\
-  "--bootnodes",\
-  "/dns/crispy.calamari.systems/tcp/30333/p2p/12D3KooWNE4LBfkYB2B7D4r9vL54YMMGsfAsXdkhWfBw8VHJSEQc", \
-  "/dns/crunchy.calamari.systems/tcp/30333/p2p/12D3KooWL3ELxcoMGA6han3wPQoym5DKbYHqkWkCuqyjaCXpyJTt", \
-  "/dns/hotdog.calamari.systems/tcp/30333/p2p/12D3KooWMHdpUCCS9j8hvNLTV8PeqJ16KaVEjb5PVdYgAQUFUcCG", \
-  "/dns/tasty.calamari.systems/tcp/30333/p2p/12D3KooWGs2hfnRQ3Y2eAoUyWKUL3g7Jmcsf8FpyhVYeNpXeBMSu", \
-  "/dns/tender.calamari.systems/tcp/30333/p2p/12D3KooWNXZeUSEKRPsp1yiDH99qSVawQSWHqG4umPjgHsn1joci" \
-]
```
Comment (ghzlatarev) [inline docker/calamari.Dockerfile]: Revert this and rename the PR to be about using the check-finalization tool for the docker test and expose 9945

Comment (Dengjianping) [reply docker/calamari.Dockerfile]: Done.

**Review by ghzlatarev — APPROVED** [review]

**Pull request merged**